webapp/home/labs/genome/annotation.yml

```diff
@@ -124,18 +124,18 @@ tabs:
                   Annotates a genome using Funannotate, includes RNAseq data with RNAstar, and protein predictions from EggNOG. <br> <br> Tools: <code>RNAstar</code> <code>funannotate</code> <code>eggnog</code> <code>busco</code> <code>jbrowse</code> <code>aegean parseval</code>
                 </p>
               inputs:
-                - datatypes:
+                - label: Genome assembly (soft-masked)
+                  datatypes:
                     - fasta
-                  label: Genome assembly (soft-masked)
-                - datatypes:
+                - label: RNAseq Illumina reads
+                  datatypes:
                     - fastq
-                  label: RNAseq Illumina reads
-                - datatypes:
+                - label: Alternative annotation
+                  datatypes:
                     - gff3
-                  label: Alternative annotation
-                - datatypes:
-                    - gbk
-                  label: Alternative annotation
+                - label: Alternative annotation
```

Comment:
Are there two "Alternative annotation" inputs, or is this one input with two accepted datatypes?

![image](https://github.com/usegalaxy-au/galaxy-media-site/assets/42562517/596eba37-6925-4c3d-b3ad-0978e6bf9aa3)

Reply:
@neoformit  there are two inputs, they are both alternative annotations